test/gie/pip.gie

```diff
@@ -0,0 +1,41 @@
+These two tests have occasionally been causing segfaults from gie, but
```

Comment:
This looks like leftovers from the debugging session. If not, it should have a more descriptive name than "pip".

Reply:
Exactly - will remove